packages/comfy-multi-player/.agents/checks/import-graph.md

```
@@ -0,0 +1,65 @@
+# Import-graph review
+
+Validate import boundaries and detect circular dependencies. This reinforces purity (KA-3, FC-3) at the module-graph level and complements `scripts/check-purity.mjs`, which reasons about the installed dependency tree and a bare-Node import of `dist/` and therefore cannot see which source module reached for what.
+
+## Steps
+
+1. `npm ci` (dependency-cruiser is a devDependency; its rules live in `.dependency-cruiser.cjs`).
+2. Run the gate:
+   ```bash
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)